scripts/api/data/metadatablocks/citation.tsv

```diff
@@ -172,14 +172,13 @@
 	language	Acatepec Me'phaa	tpx	30	tpx
```

Comment:
I'm just putting this comment on the first file...

Do we need a release note? Do we want people to reload the citation block?

Reply:
Never mind. In #10762 we already tell people to reload the citation block. This is just a bug fix on top of that PR.